scripts/create-secrets/prompts/inquirer-fuzzy-path.js

```diff
@@ -0,0 +1,100 @@
+const fs = require("fs");
+const path = require("path");
+const util = require("util");
+
+const Choices = require("inquirer/lib/objects/choices");
+const InquirerAutocomplete = require("inquirer-autocomplete-prompt");
+const stripAnsi = require("strip-ansi");
+const style = require("ansi-styles");
+const fuzzy = require("fuzzy");
+
+const readdir_ = util.promisify(fs.readdir);
+
+class InquirerFuzzyPath extends InquirerAutocomplete {
+    constructor(question, rl, answers) {
+        const rootPath = question.rootPath || ".";
+        const pathFilter = question.pathFilter || (() => true);
+        const _question = Object.assign({}, question, {
+            source: (_, pattern) => getPaths(rootPath, pattern, pathFilter)
+        });
+        super(_question, rl, answers);
+    }
+    search(searchTerm) {
```

Comment:
can we put blank lines between the methods here?

Reply:
👍